CreateItem/CreateItemWrapper.js

```diff
@@ -74,5 +76,7 @@ CreateItemWrapper.propTypes = {
   searchButtonStyle: PropTypes.string,
   marginBottom0: PropTypes.bool,
   marginTop0: PropTypes.bool,
-  stripes: PropTypes.object,
+  stripes: stripesShape.isRequired,
 };
+
+export default stripesConnect(CreateItemWrapper, { dataKey: 'createItem' });
```

Comment:
I don't see any reason to have here `stripesConnect`, do you?

Reply:
fixed